specs/order_spec.rb

```diff
@@ -111,13 +111,54 @@
       expect(order.total).must_equal before_total
     end
   end
+
+  describe "#remove_product" do
+    it "Decreases the number of products" do
+      products = { "banana" => 1.99, "cracker" => 3.00 }
```

Comment:
Good work on these tests! I feel like you've done a good job of capturing the interesting things this method does.